srcpkgs/python3-platformdirs/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'python3-platformdirs'
 pkgname=python3-platformdirs
-version=3.1.0
+version=3.1.1
```

Comment:
3.2.0 available

Reply:
Requires [hatchling>=1.13](https://github.com/platformdirs/platformdirs/commit/7ac2f5ee2fa899583f933aba87ac8825d094f29e) although there seems to be no meaningful [changes](https://github.com/pypa/hatch/releases/tag/hatchling-v1.13.0) in that version.